Makefile

```diff
@@ -259,3 +259,6 @@ check_avito_configuration:
 
 dependency_updates:
 	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) dependencyUpdates -Drevision=release
+
+benchmark_fast_check:
+	gradle-profiler --benchmark --project-dir subprojects --scenario-file gradle/performance.scenarios fastCheck
```

Comment:
```suggestion
	gradle-profiler --benchmark --project-dir subprojects --gradle-user-home ~/.gradle-user-home --output-dir build/reports/gradle-profiler  --scenario-file gradle/performance.scenarios fastCheck
```

Reply:
outputs redirect has one downside: previously it created a new directory for every new run, and after redirection it just overwrites existing results

Reply:
gradle-user-home should probably be project specific, i'am not sure though

Reply:
> and after redirection it just overwrites existing results

Why do you need multiple results? It's incorrect to compare independent runs.

Reply:
> gradle-user-home should probably be project specific

Didn't see any problems sharing it. It's like gradle GRADLE_USER_HOME

Reply:
> Didn't see any problems sharing it. It's like gradle GRADLE_USER_HOME

from doc:
```
--gradle-user-home: The Gradle user home. Defaults to <project-dir>/gradle-user-home to isolate performance tests from your other builds.
```

maybe i misinterpreted this comment, though is is a good idea to keep this home dir isolated from other projects

Reply:
> Why do you need multiple results? It's hard to compare independent runs.
> To compare two implementations we need to run two scenarios in one run.

Just running multiple checks locally, modifying parameters

Reply:
>  For now, I see only one reason to rerun - to compare different implementations \ scenarios.

yes, exactly what i'm doing

Reply:
When we have two scenarios in one gradle-profiler's report we have built-in math:
![Screenshot from 2021-06-07 20-27-20](https://user-images.githubusercontent.com/1104540/121063410-edb22e80-c7ce-11eb-9c85-38fbd938598c.png)

`97.52` - is a confidence calculated by Mann–Whitney U-test.

> I keep forgetting to clear folder before new benchmark starts

When we have multiple independent gradle-profiler results, we need to calculate the same by hand. Otherwise, it will be incorrect to compare two distributions. This is why I think it's worth overriding results, to not encourage manual comparison.